### ISSUE TITLE
Debian 11 OBS package doesn't ship `/usr/lib/rsyslog/rsyslog-rotate`

### ISSUE THREAD

**Issue by opty77** [rsyslog/rsyslog#5344]
### Expected behavior

Debian OBS package ships `/usr/lib/rsyslog/rsyslog-rotate`.

### Actual behavior

Debian OBS package doesn't ship `/usr/lib/rsyslog/rsyslog-rotate` because `debian/rsyslog.install` doesn't contain `debian/rsyslog-rotate usr/lib/rsyslog/`.

### Steps to reproduce the behavior

Download [the package](https://download.opensuse.org/repositories/home:/rgerhards/Debian_11/amd64/rsyslog_8.2402.0-1_amd64.deb) and browse its contents.

### Environment
- rsyslog version: 8.2402.0
- platform: Debian GNU/Linux 11 (Bullseye)

**Comment by davidelang** [comment]
I believe that is a script created and maintained by the Debian team, not 
something originating from the rsyslog project (I find nothing that matches in 
the git repo)

David Lang

**Comment by opty77** [comment]
I noticed Ubuntu codenames in `/usr/share/doc/rsyslog/changelog.Debian.gz`.

**Comment by opty77** [comment]
@davidelang: Yeah but OBS package uses that script (in `/etc/logrotate.d/rsyslog`).

**Comment by mbiebl** [comment]
yup, that script is part of the Debian systemd package, not something related to upstream.
That said, I dunno what "Debian OBS package" means exactly.

**Comment by opty77** [comment]
Package from [Debian Repository - rsyslog](https://www.rsyslog.com/debian-repository/).

**Comment by rgerhards** [comment]
Sorry for the very late reply. This may no longer be directly relevant to the original Debian 11 OBS package report, but I am adding the answer for people who find this issue through search while we work through the backlog. We do not lightly close old issues and we do not close them just because of age.

Current packaging has this covered. In the current tree, `packaging/ubuntu/noble/v8-stable/debian/rsyslog.install` explicitly installs `debian/rsyslog-rotate` into `usr/lib/rsyslog/`, and the logrotate configuration calls `/usr/lib/rsyslog/rsyslog-rotate`. The earlier discussion also clarified that this helper is packaging/systemd integration, not part of the upstream daemon core.

I am closing this as fixed/superseded by current packaging. If a currently supported Adiscon package still misses `rsyslog-rotate`, please open a new issue with the exact repository URL, distribution codename, package version, and `dpkg -L rsyslog` output.
